go.mod

```diff
@@ -111,3 +111,5 @@ replace (
 
 	kubevirt.io/client-go => ./staging/src/kubevirt.io/client-go
 )
+
+go 1.13
```

Comment:
Is this needed going to 1.13 have to be done carefully.

Reply:
It was auto-generated. Do you think it should be removed?

Reply:
Was auto generated by `make deps-update` ? well we should put 1.12 here but it's not related to the PR.